File: PrepedidoBusiness/Dto/Pedido/DetalhesPedido/PedidoProdutosDtoPedido.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PrepedidoBusiness.Dto.Pedido.DetalhesPedido
{
    public class PedidoProdutosDtoPedido
    {
        public string Fabricante { get; set; }
        public string NumProduto { get; set; }
        public string Descricao { get; set; }
        public short? Qtde { get; set; }
        public short? Faltando { get; set; }
        public decimal VlLista { get; set; }
        public float? Desconto { get; set; }
        public decimal? VlVenda { get; set; }
        public decimal? VlTotal { get; set; }
        public float? Comissao { get; set; }
    }
}

File: PrepedidoSpa/src/app/dto/pedido/DetalhesPedido/PedidoProdutosDtoPedido.ts
export class PedidoProdutosDtoPedido {
    Fabricante: string;
    Produto: string;
    Descricao: string;
    Qtde: number | null;
    Faltando: number | null;
    CorFaltante: string;
    CustoFinancFornecPrecoListaBase: number | null;
    Preco_Lista: number;
    Desc_Dado: number | null;
    Preco_Venda: number;
    VlTotalItem: number | null;
    VlTotalItemComRA: number | null;
    Comissao: number | null;
}
